tests/integration/logging.tcl

```diff
@@ -2,11 +2,13 @@ tags {"external:skip"} {
 
 set system_name [string tolower [exec uname -s]]
 set backtrace_supported 0
+set threads_mngr_supported 0
 
 # We only support darwin or Linux with glibc
 if {$system_name eq {darwin}} {
     set backtrace_supported 1
 } elseif {$system_name eq {linux}} {
```

Comment:
maybe we can add more systems here? i see in config.h we do support more systems, specifically BSD which we have in our CI.

Reply:
yes, the multi-thread stack trace will only be on linux, but the single thread stack trace can be tested on more platforms (reassure us that we didn't break anything)